.github/ISSUE_TEMPLATE/new-metadata-schema-request.yml

```diff
@@ -0,0 +1,53 @@
+name: New Metadata Schema Request
+description: Request a new metadata schema.
+title: "[New Metadata Schema Request]: "
+labels: ["documentation", "enhancement"]
+assignees:
+  - 
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting your new metadata schema request.
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com or GH handle
+    validations:
+      required: true
+  - type: textarea
+    id: general-description
+    attributes:
+      label: General Description
+      description: Please provide a general description of the metadata schema that you are requesting.
+      placeholder: Tell us about your metadata!
+      value: This is a metadata schema that describes the data found in a my database.
+    validations:
+      required: true
+  - type: input
+    id: review-requested-by
+    attributes:
+      label: Review Requested By
+      description: Do you have a specific reviewer that you would like to request for this schema?
```

Comment:
I kind of think we need a link to the members of the MOWG here if we're going to ask this? also 'review-requested-by' sounds like you're asking for the owner not the reviewer? I would say 'Suggested Reviewers'?

Reply:
I was thinking that in some cases owners will know specifically who they would like to review the schema and in those cases they can tag them. Since this would only happen some times, they would presumably be able to tag them here. I'm not opposed to a drop-down here but we'd have to maintain it (and make sure every MOWG member is on this GH).